octobot_evaluators/evaluator/strategy_evaluator.py

```diff
@@ -28,6 +35,13 @@ class StrategyEvaluator(AbstractEvaluator):
     def __init__(self):
         super().__init__()
         self.consumer_instance = None
+        self.strategy_time_frames = []
+        self.evaluations_last_updates = {}
+
+        # caches
+        self.available_evaluators_cache = {}
+        self.available_time_frames_cache = {}
```

Comment:
Do we need to have a cache clear method ?

Reply:
For now there is no need, you think we should add it anyway ?

Reply:
I don't know. I was thinking about reseting evaluators, do we want to also reset the strategy ? For example if a new evaluator has been dynamically added to OctoBot

Reply:
Ok, I'll add it

Reply:
Only if you think it's necessary, I was more asking than saying this should be added